RELEASE/scripts/autoscend/combat/auto_combat_default_stage3.ash

```diff
@@ -696,6 +696,12 @@ string auto_combatDefaultStage3(int round, monster enemy, string text)
 		}
 	}
 
+	//If you have tearaway pants equipped, use its skill
+	if(canUse($skill[Tear Away your Pants!]))
```

Comment:
thoughts on only doing it if monster can be staggered and we can survive a few rounds, similar to line 706 below?

Reply:
I'm actually going to move it up into the enemy_la <= 150 block above with the other IOTM skills. I don't know why I didn't put it there in the first place